indexingLogic.js

```diff
@@ -482,6 +484,24 @@ async function getBlock(block: Block) {
           console.log(
             `Successfully inserted component version record for: componentAuthorId=${componentAuthorId}, componentName=${componentName}`
           );
+
+          const versionId = versionsItems[0].id;
+          const metadataData = {
```

Comment:
If we have all the same rows as Versions (one per update), could we just add those 4 columns to Versions?

Reply:
Yes, we could have one table (versions) with additional metadata columns. 
I am also thinking of separating the fork_of (`path/to/widget@blockheight`) block height from the source path since it would be easier to count the number of times a certain widget was forked. I will push the updates today.